Description: Fix FTBFS with zlib 1.2.9 by renaming uncompress2 to
 uncompress2_internal
Origin: upstream, https://github.com/pcsxr/PCSX-Reloaded/commit/6484236cb0281e8040ff6c8078c87899a3407534
Bug-Debian: https://bugs.debian.org/897486
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/libpcsxcore/cdriso.c
+++ b/libpcsxcore/cdriso.c
@@ -1219,7 +1219,7 @@ static int cdread_sub_mixed(FILE *f, uns
 	return ret;
 }
 
-static int uncompress2(void *out, unsigned long *out_size, void *in, unsigned long in_size)
+static int uncompress2_internal(void *out, unsigned long *out_size, void *in, unsigned long in_size)
 {
 	static z_stream z;
 	int ret = 0;
@@ -1298,7 +1298,7 @@ static int cdread_compressed(FILE *f, un
 	if (is_compressed) {
 		cdbuffer_size_expect = sizeof(compr_img->buff_raw[0]) << compr_img->block_shift;
 		cdbuffer_size = cdbuffer_size_expect;
-		ret = uncompress2(compr_img->buff_raw[0], &cdbuffer_size, compr_img->buff_compressed, size);
+		ret = uncompress2_internal(compr_img->buff_raw[0], &cdbuffer_size, compr_img->buff_compressed, size);
 		if (ret != 0) {
 			SysPrintf("uncompress failed with %d for block %d, sector %d\n",
 					ret, block, sector);
